o2a/utils/template_utils.py

```diff
@@ -18,13 +18,17 @@
 import jinja2
 
 from o2a.definitions import TPL_PATH
+from o2a.utils.el_utils import escape_string_with_python_escapes
 from o2a.utils.variable_name_utils import convert_to_python_variable
 
 TEMPLATE_LOADER = jinja2.FileSystemLoader(searchpath=TPL_PATH)
-TEMPLATE_ENV = jinja2.Environment(loader=TEMPLATE_LOADER, undefined=jinja2.StrictUndefined)
+TEMPLATE_ENV = jinja2.Environment(
+    loader=TEMPLATE_LOADER, undefined=jinja2.StrictUndefined, trim_blocks=True, lstrip_blocks=True
```

Comment:
What do you think about taking this change into a separate PR?

Reply:
I needed that in order to make sure the jinja templates with proper configuration values is rendered as needed. I think it will be very difficult to split it off. 

If you know how to do it and can help with that I will be greatful.

Reply:
Please resolve this if you think it resolves the discussion @mik-laj